network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+* **MYSQL**: For MYSQL health checks, you are prompted to select the database username.
+* **PGSQL**: For PGSQL health checks, you are prompted to select the database username.
+* **LDAP**: No additional settings are required fr LDAP health checks.
+* **Redis**: No additional settings are required for REDIS health checks.
+* **TCP**: No additional settings are required for TCP health checks.
+
+### Adding server IPs (backend servers)
+
+You can add one or more IP addresses, in either IPv4 or IPv6, of the backend servers your Load Balancer's backend will forward traffic to.
+
+### Using advanced settings
+
+Here, you have the option to activate **sticky sessions**. When activated, this bind a user's session to a specific server in the pool of backend servers. This ensures that all subsequent sessions from the user are sent to the same backend server while there is at least one active session. Sticky sessions can be **cookie based** or **table based**.
+
+- **Cookie-based** stickiness uses an HTTP cookie to identify a session, and the server in the backend pool that it will stick to. For each incoming request without a stickiness cookie, the Load Balancer creates a cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. You can customize the name of the cookie.
+- **Table-based** stickiness uses the source (client) IP address to stick sessions to backends.
+
+By default, when activating sticky sessions through the console, the cookie-based method is used when HTTP protocol is selected, and the table-based method is used when TCP protocol is selected. Use the [API](https://developers.scaleway.com/en/products/lb/zoned_api/) if you want to select table-based stickiness with HTTP protocol.
+
+## Configuring a Load Balancer for SSL bridging
+
+SSL bridging allows the user to initiate a secure HTTPS connection with the Load Balancer thanks to the Load Balancer frontend's SSL certificate. The Load Balancer uses its SSL certificate to decrypt incoming HTTPS traffic. This allows the Load Balancer to carry out layer 7 actions on the received traffic. The Load Balancer's backend then initiates a new HTTPS connection to re-encrypt traffic between the Load Balancer and the backend server, this time using the backend server's certificate. 
+
+    <Lightbox src="scaleway-ssl-bridging.webp" alt="" />
+
+To configure your Load Balancer for SSL bridging:
+
+- The frontend should listen on port 443.
```

Comment:
It is not required to use port 443, you can use whichever port you want, you just need to have a certificate for your frontend.

Reply:
done